### PR TITLE
Modal and rating

### PR DESCRIPTION
**Pull request by rahulk31**
Added Modal and Rating Components.

### CODE REVIEW

File: css/components/modal.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.modal-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+  background-color: rgba(58, 49, 49, 0.7);
```
Comment (varun1233) [inline css/components/modal.css:10]: use variables

File: css/components/rating.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
+.rating-wrapper {
+  padding: 2rem 4rem;
+  border-radius: 4px;
+  box-shadow: 0 0 15px 10px rgba(0, 0, 0, 0.05);
```
Comment (varun1233) [inline css/components/rating.css]: use variable

File: css/components/modal.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+.modal-wrapper {
+  margin: auto 1rem;
+  max-width: 65rem;
+  padding: 3rem 4rem;
+  text-align: center;
+  background-color: var(--white);
+  border-radius: 9px;
+}
+
+.modal-wrapper p {
```
Comment (j24m) [inline css/components/modal.css]: This is fine, but you can use class for p tag too, to leave no place for mistakes. :)

**Review by j24m — APPROVED** [review]
Looks good to merge and the UI looks beautiful. :)

File: css/components/modal.css (hunk 3 above)
Comment (rahulk31) [reply css/components/modal.css]: Thanks @j24m for the review. I have made suggested changes. 😄

File: css/components/rating.css (hunk 2 above)
Comment (rahulk31) [reply css/components/rating.css]: Thanks Varun for the review, I have made the suggested changes. 😄

File: css/components/modal.css (hunk 1 above)
Comment (rahulk31) [reply css/components/modal.css:10]: Thanks Varun for the review, I have made the suggested changes. 😄

**Pull request merged**